### PR TITLE
Add published vendor translation

### PR DESCRIPTION
**Pull request by roseffendi**
This PR include published vendor translation at generation.
I think it will come handy to generate the published package translation all at once.

### CODE REVIEW

**Comment by martinlindhe** [comment]
If this is useful, then sure we can merge it. But I want it behind a command line switch, and disabled by default. The reason is that scanning the vendor folder might add quite some time when running this package (if the vendor folder is big).

You can see how command line switches is currently handled in https://github.com/martinlindhe/laravel-vue-i18n-generator/blob/master/src/Commands/GenerateInclude.php

**Comment by roseffendi** [comment]
Yes, i can make command line switch for that.
How would you like it to be formatted ?

Currently it's like this.

    [en] => [
	    "yes" => "Yes",
	    "no" => "No"
	    "vendor" => [
		    "package-hint-one" => [
			    "package-one-lang" => "Translation"
		    ],
   		    "package-hint-two" => [
			    "package-two-lang" => "Translation"
		    ],
	    ]
    ]

File: tests/GenerateTest.php [hunk 1]
```diff
@@ -6,7 +6,7 @@ class GenerateTest extends \PHPUnit_Framework_TestCase
 {
     private function generateLocaleFilesFrom(array $arr)
     {
-        $root =  sys_get_temp_dir() . '/' . sha1(microtime(true) . mt_rand());        
+        $root =  __DIR__ . '/tmp/' . sha1(microtime(true) . mt_rand());
```
Comment (martinlindhe) [inline tests/GenerateTest.php]: In all great work, just one last little detail!

Is this change required?

It may be troublesome since it would require the user to have write access to this directory.

Given it's in a test, it might not be an issue. What are your thoughts?

**Comment by martinlindhe** [comment]
The formatting looks good, thank you for your work on this!

File: tests/GenerateTest.php (hunk 1 above)
Comment (roseffendi) [reply tests/GenerateTest.php]: Nah, it just my case at for #36.
My bad, i forgot to change it back.

**Comment by martinlindhe** [comment]
All good, thanks alot for this!
